applications/resturant/endpoints.py

```
@@ -436,6 +441,15 @@
     serializer_class = MenuSerializer
     permission_classes = [IsAuthenticated]
 
+    def post(self, request):
```

Ruff (0.11.9)

Missing return type annotation for public function `post`

(ANN201)